src/components/atoms/navbar.tsx

```diff
@@ -88,7 +88,7 @@ export const NavBar: FC<NavBarProps> = ({
               accessibilityRole="button"
               accessibilityHint={t('navbar:backHint')}
               onPress={() => navigation.goBack()}>
-              <View style={styles.back}>
+              <View hitSlop={{left: 10, right: 20}} style={styles.back}>
```

Comment:
@JohnAGill I would make left 12 (till the screen margin), right 12 as well just to keep is consistent and should be also add top & bottom 8? (just like the padding on the container?)